docs/howto/garbage-collection.md

```diff
@@ -282,56 +282,78 @@ repositories may require increasing a read timeout.  If you run into timeout err
 GC was created to hard-delete objects from your underlying objects store according to your retention rules. However, when you start
 using the feature you may want to first gain confidence in the decisions GC makes. The GC backup and restore utility helps you do that. 
 
-This utility is a Spark application that uses [distCp](https://hadoop.apache.org/docs/current/hadoop-distcp/DistCp.html) under the hood to copy objects marked by GC as expired from one location to another. 
-
 **Use-cases:**
 * Backup: copy expired objects from your repository's storage namespace to an external location before running GC in [sweep only mode](#sweep-only-mode).  
 * Restore: copy objects that were hard-deleted by GC from an external location you used for saving your backup into your repository's storage namespace.
 
+### Using rclone
+
+Follow [rclone documentation](https://rclone.org/docs/) to configure remote access to the underlying storage used by lakeFS.
+Replace `LAKEFS_STORAGE_NAMESPACE` with remote:bucket/path points to the lakeFS repository storage namespace.
+The `BACKUP_STORAGE_LOCATION` attribute points to a location for backing up the objects that will be deleted.
+
+#### Backup command
+
+   ```shell
+   rclone --include "*.txt" cat "<LAKEFS_STORAGE_NAMESPACE>/_lakefs/retention/gc/addresses.text/mark_id=<MARK_ID>/" | \
```

Comment:
This path is true for committed GC but it's not correct for uncommitted.
Same thing for the restore command

Reply:
NM, I see it's in a different section